### PR TITLE
Add tests for new relic matches and other updates

### PR DESCRIPTION
**Pull request by alain-hoang**
* Add condition and test to not match newrelic keys in aws rules
* Fix variables for consistency

### CODE REVIEW

File: test/seekrets.bats [hunk 1]
```diff
@@ -60,6 +60,12 @@ load test_helper
     [ $status -gt 0 ]
 }
 
+@test "git-seekrets does not find newrelic keys as aws keys in test repo" {
```
Comment (LinuxBozo) [inline test/seekrets.bats]: How does this test that it didn't find a newrelic key as AWS key exactly? You add a file with a new relic secret, and then, you get a result of 1 secret found. I see nothing related to AWS in this test.

File: seekret-rules/aws.rule [hunk 2]
```diff
@@ -1,8 +1,12 @@
 secret_access_key:
   match: (\"|')?(AWS|aws|Aws)?_?(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9/\+=]{40}(\"|')?
+  unmatch:
+    - (\"|')?(NEW|New|new)_?(RELIC|Relic|relic)_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9/\+=]{40}(\"|')?
 
 access_key_id:
   match: (\"|')?(AWS|aws|Aws)_?(ACCESS|access|Access)?_?(KEY|key|Key)_?(ID|id|Id)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
+  unmatch:
+    - (\"|')?(NEW|New|new)_?(RELIC|Relic|relic)_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
```
Comment (LinuxBozo) [inline seekret-rules/aws.rule]: Pretty sure this is doing more than it has to.. I should only have to find some combination of "new" and "relic".

Comment (rogeruiz) [inline seekret-rules/aws.rule]: I agree @alain-hoang this check just needs to be simpler. Something along the lines of the check below.

```yaml
unmatch:
  - .+(RELIC|Relic|relic).+
```

**Comment by rogeruiz** [comment]
Once these tests pass this will be merged in. 💯

File: test/seekrets.bats [hunk 3]
```diff
@@ -60,9 +60,9 @@ load test_helper
     [ $status -gt 0 ]
 }
 
-@test "git-seekrets does not find newrelic keys as aws keys in test repo" {
```
Comment (LinuxBozo) [inline test/seekrets.bats]: Wait, I thought the issue was that a new relic key was tripping the AWS rule, hence the unmatch needed in the AWS rule?

Comment (rogeruiz) [inline test/seekrets.bats]: Yup, I reverted it. Thanks 💯

**Comment by rogeruiz** [comment]
We'll need to rebase this with #117 to get tests passing again.

**Pull request merged**